ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -297,6 +298,14 @@ data instance BlockQuery (ShelleyBlock proto era) :: Type -> Type where
   -- longer used (because mainnet has hard-forked to a newer version), it can be
   -- removed.
 
+  GetAccountState 
+    :: ( CG.ConwayEraGov era
+       , LC.EraTxOut era
+       , Show (StashedAVVMAddresses era)
+       , Eq (StashedAVVMAddresses era)
```

Comment:
Do we need these constraints?

Reply:
All of them became redundant when I got rid of `NewEpochState`